### PR TITLE
[git-webkit] Handle local commit on production branch

### PR DESCRIPTION
**Pull request by JonWBedard**
#### 5b14e1658e477ccf952d532fe671f71b951004c0
<pre>
[git-webkit] Handle local commit on production branch
<a href="https://bugs.webkit.org/show_bug.cgi?id=235974">https://bugs.webkit.org/show_bug.cgi?id=235974</a>
&lt;rdar://problem/88346615 &gt;

Reviewed by NOBODY (OOPS!).

* Tools/Scripts/libraries/webkitscmpy/setup.py: Bump version.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/__init__.py: Ditto.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/mocks/local/git.py: Handle moving
branches to remote reference.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:
(PullRequest.main): Move local branch ref to remote reference.
* Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/pull_request_unittest.py:
</pre>

### CODE REVIEW

File: Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py [hunk 1]
```diff
@@ -93,6 +93,11 @@ def create_commit(cls, args, repository, **kwargs):
 
         return 0
 
+    @classmethod
+    def move_production_ref_to_brach_point(cls, args, repository):
```
Comment (aj062) [inline Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:97]: Typo: brach => branch

**Comment by aj062** [comment]
looks good on high level

File: Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py (hunk 1 above)
Comment (JonWBedard) [reply Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py:97]: Oops!

**Comment by JonWBedard** [comment]
> looks good on high level

Still need to add the program logic

**Comment by JonWBedard** [comment]
This PR was being used to demonstrate some tooling, #120 has the actual change

**Pull request closed without merging**